webui/.gitignore

```diff
@@ -0,0 +1,31 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+.DS_Store
+dist
+dist-ssr
+coverage
+*.local
+
+/cypress/videos/
+/cypress/screenshots/
```

Comment:
what are these? can you add comments for the sections please?

Reply:
@mr-tz these entries are automatically added when I generated the .gitignore - we don't use this tool, so I removed them.